hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionPlugin.java

```diff
@@ -28,9 +31,15 @@ private void createConvention(Project target) {
         .create(JavaConventionExtension.EXTENSION_NAME, JavaConventionExtension.class);
   }
 
+  private void configureCompatibility(Project target) {
+    Property<JavaVersion> javaVersion = javaConventionExtension(target).compatibilityVersion;
+    javaPluginExtension(target).setTargetCompatibility(javaVersion);
```

Comment:
bug here fixed in follow up - https://github.com/hypertrace/hypertrace-gradle-java-convention-plugin/pull/3